server/expression.js

```diff
@@ -26,22 +26,32 @@ expression.RawMatrix;
 
 /**
  * @typedef {{
+ *   allGeneNames: !Array<string>,
+ *   allConditionNames: !Array<string>,
+ *   allValueMin: number,
+ *   allValueMax: number
+ * }}
+ */
+expression.MatrixAll;
+
+/**
+ * @typedef {{
```

Comment:
Please update wiki for changed queries.
https://github.com/ViDA-NYU/genotet/wiki/Expression-Queries